planner/core/preprocess.go

```diff
@@ -258,8 +259,8 @@ func (p *preprocessor) checkBindGrammar(originNode, hintedNode ast.StmtNode) {
 			return
 		}
 	}
-	originSQL := parser.Normalize(originNode.Text())
-	hintedSQL := parser.Normalize(hintedNode.Text())
+	originSQL := parser.Normalize(utilparser.RestoreWithDefaultDB(originNode, defaultDB))
+	hintedSQL := parser.Normalize(utilparser.RestoreWithDefaultDB(hintedNode, defaultDB))
```

Comment:
This change would relax the restriction for CREATE BINDING / DROP BINDING? IMHO this change is unnecessary for this PR?

Reply:
This change is to rewrite SQL with default DB. I did not find a better way to do this.
I think this relaxation of restrictions is acceptable, and it will not have any particular impact in fact. This is the reason why I did this.
Is there anything I haven't considered for this relaxation?